src/interfaces.ts

```diff
@@ -58,3 +59,21 @@ export interface INamedVector {
   name: string
   vector: Vector
 }
+
+/**
+ * Interface for localized operators
+ */
+export interface ILocalOperator {
+  x: number
+  y: number
+  operator: Operator
+}
+
+/**
+ * Interface for localized operators
+ */
+export interface ITileIntensity {
+  x: number
+  y: number
+  probability: number
+}
```

Comment:
Nice and needed

Reply:
Will use the IXYOperator interface definition since it's already present.